docs/changelog.md

```diff
@@ -1,4 +1,6 @@
 # TBD
+### Features
+* Added a new method `CheckServiceAvailability` in network context, it can be used to check if a service is running well
```

Comment:
s/CheckServiceAvailability/WaitForEndpointAvailability/

Reply:
Recommended rephase: "Added a new method `WaitForEndpointAvailability` in NetworkContext that can be used to wait until a service's endpoint becomes available"